### PR TITLE
Docker: Mon-stack Version update

### PR DESCRIPTION
**Pull request by pranavv0**
Mon-stack Update:
 1. Grafana: 7.2.2 to 9.191 (https://logmanager-docker.iudx.io/)
     Loki: 2.0.0 to 2.6.1
     Promtail: 2.0.0 to 2.6.1
     Prometheus: Updated to 2.38(Latest)
2. Update in Stack file
3. Updated grafan/Dockerfile
4. Updated grafana/Readme

### CODE REVIEW

File: Docker-Swarm-deployment/single-node/monitoring-stack/mon-stack.yaml [hunk 1]
```diff
@@ -20,17 +21,9 @@ services:
...
-    logging:
-      driver: "json-file"
-      options:
-        max-file: "3"
-        max-size: "5m"
-        tag: "{\"name\":\"{{.Name}}\",\"id\":\"{{.ID}}\"}"
 
   prometheus:
-    image: prom/prometheus:v2.26.0
+    image: prom/prometheus
```
Comment (karun-singh) [inline Docker-Swarm-deployment/single-node/monitoring-stack/mon-stack.yaml:26]: Mention the image version/tag

File: Docker-Swarm-deployment/single-node/monitoring-stack/mon-stack.yaml [hunk 2]
```diff
@@ -135,18 +112,10 @@ services:
...
-     - ./secrets/.grafana.env 
-    cap_drop:
-      - ALL
-    logging:
-      driver: "json-file"
-      options:
-        max-file: "3"
-        max-size: "5m"
-        tag: "{\"name\":\"{{.Name}}\",\"id\":\"{{.ID}}\"}"
+     - ./.grafana.env
```
Comment (karun-singh) [inline Docker-Swarm-deployment/single-node/monitoring-stack/mon-stack.yaml:115]: Keep this env file in `secrets/` directory itself. Or update in the README secrets directory structure.

**Review by abhi4578 — CHANGES_REQUESTED** [review]
Please add software update changes on top of the refactor done in #156 . please see the commits
1. merge node exporter ansible to infra provision : https://github.com/datakaveri/iudx-deployment/pull/156/commits/5c2ef409fb9fcf3e4b29d5c2c4954b37468c34c0
    - move mon-stack/ansible to infrastructure dir
3. refactor mon-stack: https://github.com/datakaveri/iudx-deployment/pull/156/commits/aa4a2bd7b104fba0e3e8cff2b136a0987cecf5f8
    - simplify deployment
    - add resources, custom stack file
    - remove certs, as its no longer required, TLS through nginx
    - '.yml' files to '.yaml' files

- also update node-exporter, blackbox


@pranavv0

**Comment by abhi4578** [comment]
Better to close this PR and add the changes in new one

**Comment by pranavv0** [comment]
Continued in another PR

**Pull request closed without merging**